content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -7,19 +7,31 @@ This document describes how to install SODA projects in a local cluster with det
 
 `Hereafter, Hotpot refers to API, Controller and Dock projects`
 
-### Pre-config (Ubuntu 16.04 or Ubuntu 18.04)
-All the installation work is tested on `Ubuntu 16.04` and `Ubuntu 18.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+{{% notice info %}}
+**Ubuntu 16.04 support for SODA is deprecated as of SODA Jerba release v1.4.0.** <br />
+**If you have a requirement to install SODA on Ubuntu 16.04, please contact us on slack and we will try to help with the setup.**
+{{% /notice %}}
+
+{{% notice info %}}
+**Ubuntu 20.04 support for SODA is in the pipeline and will be GA in the next SODA release.** <br />
+**If you have a requirement to install SODA on Ubuntu 20.04, please contact us on slack and we will try to help with the setup.**
```

Comment:
When its planned for next rel, why special help required.

Reply:
Someone who is using the latest version of Ubuntu and does not have the option to downgrade or use virtual box may need help with certain steps that may fail. 
This is just to inform them to contact us so we can help with the setup.
This may also help with any issues that we may not always catch with installations on new environments.